tests/test_endpoints.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+import json
+from app import create_app
+import sys
+sys.path.append("..")
+
+
+@pytest.yield_fixture
+def app():
+    app = create_app("testing")
+    yield app
+
+
+@pytest.fixture
+def test_cli(loop, app, sanic_client):
+    return loop.run_until_complete(sanic_client(app))
+
+
+async def test_index(test_cli):
+    resp = await test_cli.get('/')
+    assert resp.status == 200
+
+
+async def test_getLanguages_with_no_query_param(test_cli):
+    resp = await test_cli.get('/languages')
+    resp_json = await resp.json()
+    assert resp_json["total_results"] == 0
```

Comment:
Good job asserting that there is actually something in the response here. I do have few questions:
- Does calling /languages endpoint always return 0 results?
- Is there a way to test the case where more than 0 results are returned? Ideally, the test should include the test for pagination as well.

Reply:
Right now, the request is hitting an empty database. So make sense that the endpoint returns 0 results. I will add dummy data for test.